components/purchase_modal/purchase_modal.tsx

```diff
@@ -290,6 +318,72 @@ export default class PurchaseModal extends React.PureComponent<Props, State> {
         });
     }
 
+    paymentFooterText = () => {
+        const normalPaymentText = (
+            <FormattedMessage
+                defaultMessage={'Payment begins: {beginDate}'}
+                id={'admin.billing.subscription.paymentBegins'}
+                values={{
+                    beginDate: getNextBillingDate(),
+                }}
+            />
+        );
+
+        let payment = normalPaymentText;
+        if ((!this.props.isFreeTrial && this.state.currentProduct?.name.toLowerCase().includes('starter')) &&
+                !this.state.selectedProduct?.name.toLowerCase().includes('starter')) {
+            const announcementTooltip = (
+                <Tooltip
+                    id='proratedPayment__tooltip'
+                    className='proratedTooltip'
+                >
+                    <div className='tooltipTitle'>
+                        <FormattedMessage
+                            defaultMessage={'Prorated Payments'}
+                            id={'admin.billing.subscription.proratedPayment.tooltipTitle'}
+                        />
+                    </div>
+                    <div className='tooltipText'>
+                        <FormattedMessage
+                            defaultMessage={'If you upgrade to {selectedProductName} from Cloud Starter mid-month, you will be charged a prorated amount for both plans.'}
```

Comment:
Can't a an upgrade happen during anytime within the month and not necessarily mid-month? I think @annelieseklein could help clarify on this one.

Reply:
yup! anytime within the month @AGMETEOR @pablovelezvidal

Reply:
I think mid-month is the expression to refer `in the middle of the month` which not exactly refers to the literal middle/half. Is my assumption correct @annelieseklein ?